helix.go

```diff
@@ -118,9 +119,14 @@ func NewClient(options *Options) (*Client, error) {
 		options.HTTPClient = http.DefaultClient
 	}
 
+	baseURL := APIBaseURL
+	if options.APIBaseURL != "" {
+		baseURL = options.APIBaseURL
+	}
+
 	client := &Client{
 		opts:    options,
-		baseURL: APIBaseURL,
+		baseURL: baseURL,
```

Comment:
TBH I don't know why we have this option. I don't think it even gets used. Looks like it may have been missed during a previous refactor.

It probably be removed, and instead we can rely on the new `options.APIBaseURL` property you've introduced.

Reply:
Yes, seems there are no references to this field except this one, but then the field can also be removed from the ``Client`` struct